src/Psalm/Internal/Analyzer/Statements/Expression/BinaryOp/ArithmeticOpAnalyzer.php

```diff
@@ -733,7 +733,7 @@ private static function analyzeOperands(
                         }
                     } else {
                         $result_type = Type::combineUnionTypes(
-                            $always_positive ? Type::getPositiveInt(true) : Type::getInt(true),
+                            $always_positive ? Type::getPositiveInt(true): Type::getInt(true),
```

Comment:
Was it forced by slevomat CS?

Reply:
No, I’m afraid, it was me. Don’t drink and code :smirk:

Reply:
There were actually several of those.

Reply:
And yet again you are right